### ISSUE TITLE
Pretrained weight for semantic segmentation 

### ISSUE THREAD

**Issue by leotaku98** [aladdinpersson/Machine-Learning-Collection#158]
Is there a pre-trained weight for the image segmentation task? Starting from zero results in an undesirable accuracy for me:(
